src/slick.grid.ts

```diff
@@ -4658,15 +4718,18 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
 
     const x = document.createElement('div');
     const xRight = document.createElement('div');
-    x.innerHTML = this.sanitizeHtmlString(stringArrayL.join(''));
-    xRight.innerHTML = this.sanitizeHtmlString(stringArrayR.join(''));
-    console.time("applying css");
-    const elements1 = x.querySelectorAll('[data-top]') as NodeListOf<HTMLElement>;
-    const elements2 = xRight.querySelectorAll('[data-top]') as NodeListOf<HTMLElement>;
-    this.applyTopStyling(elements1);
-    this.applyTopStyling(elements2);
-    console.timeEnd("applying css");
-    
+    if (this._options.nonce) {
```

Comment:
same as before I don't want to keep 2 implementations to support unless there's a big perf hit which I don't think there is from previous discussions. Have you tried the `autoHeight` grid yet?

Reply:
Interesting tried to add autoHeight: true with 50.000 rows
tho with nonce seems to be faster than the non nonce version, around 1/3 faster sometimes 1/2 faster

ex a search without nonce took 400ms where with nonce it took 200ms

Initial load took 1500 ms without nonce and 1000 ms with nonce (first execution)
second run took 1300 ms without nonce down to 900 with nonce (second execution)

not exactly sure why on initial load it gets called twice tho this might be a concern (not sure current prod does this, havent checked)

Reply:
yeah I can see that the double doesn't make much difference 1 vs 2ms isn't a big deal I guess, but when you add them up in the `autoHeight`, 400ms is definitely noticeable. I'm not sure what to do at this point but I feel the worst case should always be considered before making a decision and `autoHeight` is definitely the worst case usage I assume. If we can find a way to remove that double call of `renderRows` then perhaps it might be more acceptable to go with the nonce. I'm not sure what to go with at this point...

Reply:
Think i might have misunderstood my text, new code is faster than the old version
first set of times is ```without nonce``` and second time is ```with nonce```

Reply:
oh I got them reversed indeed, if that's the case then yeah let's keep only new code (with nonce) since that will help with CSP compliance. 

> Hmm interesting the dataView code even on prod requests renderRows twice.

I'm still curious to see if you can find why it does that, but I think that could, and probably should, be a separate PR

Reply:
I've commited the cleanup now and everything as far as i know should be in this PR